packages/gestalt/src/Badge.js

```diff
@@ -3,20 +3,29 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 import styles from './Badge.css';
+import colors from './Colors.css';
 
 type Props = {|
   position?: 'middle' | 'top',
   text: string,
+  color?: 'blue',
```

Comment:
@denamwangi since we're changing the default treatment, can we remove this extra prop?

Reply:
Sure!